src/checkers/inference/InferenceMain.java

```diff
@@ -298,7 +298,7 @@ private InferrableChecker getRealChecker() {
         if (realChecker == null) {
             try {
                 realChecker = (InferrableChecker) Class.forName(
-                        InferenceOptions.checker, true, ClassLoader.getSystemClassLoader()).newInstance();
```

Comment:
All changes on `InferenceMain.java `, `CnfVecIntSerializer.java`, and `JsonDeserializer.java` have already been merged into typetools, check https://github.com/typetools/checker-framework-inference/commit/65aa1cac6df1f9ab20cc75030292c9bde4936fce